tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -34,7 +45,12 @@ sub get_script_run {
     handle_sp_in_settings_with_fcs("GUEST_LIST");
     my $guest_list = get_required_var("GUEST_LIST");
 
-    $pre_test_cmd = "$pre_test_cmd -p $product_upgrade -r $product_upgrade_repo -t $max_test_time -g \"$guest_list\"";
+    $pre_test_cmd = "$pre_test_cmd -p $product_upgrade -r $product_upgrade_repo -g \"$guest_list\"";
+    if (is_s390x) {
+        $pre_test_cmd .= "  | tee /tmp/guest_upgrade_test.log";
```

Comment:
How about s390x_guest_upgrade_test ?

Reply:
Because of there was just only called `test_full_guest_upgrade.sh` shell script to trigger s390x guest upgrade test, was included one time ctcs2, means that missed required test log **guest_upgrade_test.log** here.
Thus, add this missed  guest_upgrade_test.log for osd s390x guest upgrade test.